packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -7,13 +7,62 @@
     outline: none;
 }
 
-.hop-Popover__dialog {
+.hop-Popover {
+    --hop-Popover-max-inline-width: calc(100% - (var(--container-padding) * 2)); /* --container-padding is set in Popover.tsx */
+    --hop-Popover-inline-width: 25rem;
+    --hop-Popover-dropdown-inline-width: var(--trigger-width);
+    --hop-Popover-dropdown-auto-inline-width: auto;
+
+    /* Internal Variables */
+    --width: var(--hop-Popover-inline-width);
+    --max-width: var(--hop-Popover-max-inline-width);
+    
+    inline-size: var(--width);
+    max-inline-size: var(--max-width);
+}
+
+.hop-Popover[data-trigger="Select"] {
+    --width: var(--hop-Popover-dropdown-inline-width);
+}
+
+.hop-Popover[data-auto-width] {
+    --width: var(--hop-Popover-dropdown-auto-inline-width);
+}
+
+.hop-Popover[data-entering] {
+    animation: fade 0.25s;
+}
+
+.hop-Popover[data-exiting] {
+    animation: fade 0.25s reverse;
+}
+
+@keyframes fade {
+    from {
+        opacity: 0;
+    }
+
+    to {
+        opacity: 1;
+    }
+}
+
+.hop-Popover__dialog,
+.hop-Popover__dropdown {
     --hop-Popover-background: var(--hop-neutral-surface);
     --hop-Popover-font-family: var(--hop-body-sm-font-family);
     --hop-Popover-font-size: var(--hop-body-sm-font-size);
     --hop-Popover-font-weight: var(--hop-body-sm-font-weight);
     --hop-Popover-line-height: var(--hop-body-sm-line-height);
     --hop-Popover-color: var(--hop-neutral-text);
+    --hop-Popover-grid-template: 
+                            "header header"
+                            "content content" 1fr
+                            "footer action"
+                            / auto auto;
```

Comment:
I'm used to see `.` and not `/` are they the same?

Reply:
. usually means that nothing should be in the first column. In this case, everything after / is defining the size of the columns. I didn't want to do the shorthand, but the linter complains when I define the properties separatly.